package.json

```diff
@@ -13,8 +13,9 @@
     "/src",
     "/scss"
   ],
+  "sideEffects": false,
   "scripts": {
-    "build": "webpack --mode production --display-modules && node build-scss.js",
+    "build": "rm -rf ./dist && ./node_modules/.bin/babel src --out-dir dist --source-maps --ignore **/*.test.jsx,**/*.stories.jsx,**/__mocks__,**/setupTest.js && node build-scss.js",
```

Comment:
Note all the exclusions - this means just the code will be included.

Reply:
is `./node_modules/.bin/babel` needed instead of just `babel`?